docs/user-guide/resource-browser.md

```diff
@@ -1,71 +1,438 @@
 # Resource Browser
 
-`Resource Browser` lists all of the resources running in your cluster. You can use it to view, inspect, manage, and delete resources in your cluster. You can also create resources from the `Resource Browser`.
+## Introduction
 
-Resource Browser are helpful for troubleshooting issues. It supports multi-cluster.
+The Devtron Resource Browser provides you a central interface to view and manage all your [Kubernetes objects](../../reference/glossary.md#objects) across clusters.  It helps you perform key actions like viewing logs, editing live manifests, and even creating/deleting resources directly from the user interface. This is especially useful for troubleshooting purposes as it supports multi-cluster too.
```

Comment:
We have a few good blogs on resource browser, should we link it here somewhere? Why should one use resource browser over kubectl type blogs?

Reply:
Added